jellyfin-model/src/main/kotlin-generated/org/jellyfin/sdk/model/api/ConfigurationPageInfo.kt

```diff
@@ -46,7 +46,7 @@ public data class ConfigurationPageInfo(
 	@SerialName("DisplayName")
 	public val displayName: String? = null,
 	/**
-	 * Gets or sets the type of the configuration page.
+	 * Enum ConfigurationPageType.
```

Comment:
Should be "Gets or sets the type of the configuration page.":

https://github.com/jellyfin/jellyfin/blob/release-10.7.z/Jellyfin.Api/Models/ConfigurationPageInfo.cs#L72

But is "Enum ConfigurationPageType." from here:

https://github.com/jellyfin/jellyfin/blob/release-10.7.z/MediaBrowser.Controller/Plugins/IPluginConfigurationPage.cs#L37


Is this an issue in Swashbuckle or did we configure something wrongly @crobibero?